docs/configuration.md

```diff
@@ -16,7 +16,7 @@ You should place the file in the root of your upstream repo. Packit accepts thes
  Key name                  | type            | description
 ---------------------------|-----------------|----------------------------------------------------------------------
  `specfile_path`           | string          | relative path to a spec file within the upstream repository
- `synced_files`            | list of strings | a list of relative paths to files in the upstream repo which are meant to be copied to dist-git during an update
+ `synced_files`            | list of dicts   | a list of a dictionaries with relative paths to files in the upstream repo which are meant to be copied to dist-git during an update
```

Comment:
This is a giant no from my side: do you really want to update every single packit.yaml out there?

Reply:
ok, then we will have `synced_files` for single files and `sync` for `src` and `dest` option.

Reply:
yes please, afaik it's also written like this in the original issue